HLTrigger/Configuration/python/Tools/confdb.py

```diff
@@ -503,7 +503,9 @@ def overrideOutput(self):
   def addEras(self):
     if self.config.eras is None:
       return
-    self.data = re.sub(r'process = cms.Process\( *"\w+"', 'from Configuration.StandardSequences.Eras import eras\n\g<0>, '+', '.join('eras.' + era for era in self.config.eras.split(',')), self.data)
```

Comment:
@Martin-Grunewald could you please check?  It looks to me just a reorganization of the code 
@kpedro88 in this case you do not avoid the import, just split the syntax into pieces, is it really an improvement?

Reply:
I tested earlier that my change produces the same output.

This file itself does still import Eras.py, but the generated config does not (unless I misunderstand how this is used elsewhere).

Reply:
This part seems currently not used, so...

Reply:
..but where are the import statements for the various eras in the generated file?

Reply:
https://github.com/cms-sw/cmssw/pull/26590/files#diff-65d8bdcc7d5f19b47d1029875255fec0R508
the imports are obtained from `eras.pythonCfgLines`, which now stores this information

Reply:
OK, thanks!